cam:
  width: 640
  height: 480
  K: [540.0, 0.0, 320.0, 0.0, 540.0, 240.0, 0.0, 0.0, 1.0]
  topic_name: /camera/depth/image_raw
  frame_id: camera_depth_optical_frame

tsdf_node:
  frame_id: task_frame
  size: 0.32
  publish_rate: 2

grasp_detector:
  model_path: /home/michel/Developer/catkin_ws/src/grasp_playground/vgn/data/runs/Feb26_11-23-36,net=conv,data=urdf_zoo,augment=True,batch_size=32,lr=3e-04,descr=/vgn_conv_40.pth
